cypress/e2e/tests/global-ui/favicons.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { LoginPagePo } from '@/cypress/e2e/po/pages/login-page.po';
 
-describe('Global UI', () => {
+describe('Global UI', { tags: ['@adminUser', '@standardUser'] }, () => {
```

Comment:
there's some debate whether we need to run some tests as both... or just one will do

Reply:
Sounds like maybe we need to come up with a process for how to determine which tests run against both or just one. Is this something we should discuss in team-container-ui?

Reply:
Maybe we see how long they take once merged. i noticed the 1h10 was an outlier and other runs were quicker